start-from-zero/Javascript/src/views/evaluation/evaluation.js

```diff
@@ -1,145 +1,144 @@
+    
+        const localSt = localStorage.length;
+        var rows;
+        if(localSt == 1){
+            rows = [];
+        }else{
+            rows = JSON.parse(localStorage.getItem("localArr"));
+        }
+    
+    
+        this.render = `
+        ${windowObjectWrap.NAV()}
+        ${EvaluationsTable({
+            items: rows,
+            itemHeadings: headings
+        })}
+        ${windowObjectWrap.FooterFixed()}
+        `;
+    
     }
-
-
-    this.render = `
-    ${windowObjectWrap.NAV()}
-    ${windowObjectWrap.EvaluationsTable({
-        items: rows,
-        itemHeadings: headings
-    })}
-    ${windowObjectWrap.FooterFixed()}
-    `;
-}
-
-windowObjectWrap.EvaluationTableHeader = function(options){
-    const headings = [];
-    options.headings.map((el)=>{
-        const generatedEl = `<th class='table-head-element'>${el}</th>`;
-        headings.push(generatedEl);
-    });
-    const headingsEl = `${headings.join('')}`;
-    return `
-    <thead>
-        <tr class="table-head-row">
-            ${headingsEl}
+    
+    const EvaluationTableHeader = function(options){
+        const headings = [];
+        options.headings.map((el)=>{
+            const generatedEl = `<th class='table-head-element'>${el}</th>`;
+            headings.push(generatedEl);
+        });
+        const headingsEl = `${headings.join('')}`;
+        return `
+        <thead>
+            <tr class="table-head-row">
+                ${headingsEl}
+            </tr>
+        </thead>
+        `;
+    }
+    
+    const EvaluationTableRow = function(options={}){
+        return `
+        <tr class='table-body-tr' data-id='${options.id}'>
+            <td>${options.username}</td>
+            <td>${options.date}</td>
+            <td>${options.radio}</td>
+            <td class='detalii'>Detalii 
+                <div class="button-toggle">
+                    <span class="plus-button"></span>
+                </div>
+            </td>
+            
         </tr>
-    </thead>
-    `;
-}
-
-windowObjectWrap.EvaluationTableRow = function(options={}){
-    return `
-    <tr class='table-body-tr'>
-        <td>${options.username}</td>
-        <td>${options.regex}</td>
-        <td>${options.radio}</td>
-        <td class='detalii'>Detalii 
-            <div class="button-toggle">
-                <span class="plus-button"></span>
-            </div>
-        </td>
+        `;
         
-    </tr>
-    `;
-}
-
-windowObjectWrap.EvaluationTableBody = function(options={}){
-    const rowsElements = options.items.map(function(rowObj){
-        return windowObjectWrap.EvaluationTableRow(rowObj);
-    });
-    const rowsEl = rowsElements.join('');
-    return `
-    <tbody class='table-body'>
-        ${rowsEl}
-    </tbody>
-    `;
-};
+    }
 
-windowObjectWrap.DetailsEvaluation = function(options={}){
+    const EvaluationTableBody = function(options={}){
+        const rowsElements = options.items.map(function(rowObj){
```

Comment:
What do you think would happen if, instead of passing in a callback that invokes `EvaluationTableRow` to `optiona.items.map`, you'd pass it `EvaluationTableRow` directly?